contrib/nydusify/cmd/nydusify_test.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2023 Alibaba Cloud. All rights reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIsPossibleValue(t *testing.T) {
+	value := "qwe"
+	list := []string{"abc", "qwe", "xyz"}
+	if !isPossibleValue(list, value) {
+		t.Errorf("isPossibleValue() expected return true, but got false")
+	}
+
+	// Failure situation
+	value2 := "vdf"
+	if isPossibleValue(list, value2) {
+		t.Errorf("isPossibleValue() expected return false, but got true")
+	}
+}
+
+func TestAddReferenceSuffix(t *testing.T) {
+	source := "localhost:5000/nginx:latest"
+	suffix := "-suffix"
+	target, err := addReferenceSuffix(source, suffix)
+	if err != nil {
```

Comment:
How about using `require.NoError(t, err)`, this is much simpler.

Reply:
Fixed.